matRad_calcParticleDoseBixel.m

```diff
@@ -34,23 +36,45 @@
 %
 % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+% simplification since SSD is not exactly in y direction but sourceRashiDistance is
+% upstream value is used, since we then don't rashi need geometrical thickness
+rashiSurfaceDist = SSD - rangeShifter.sourceRashiDistance;
+
 % range shift
 depths = baseData.depths + baseData.offset;
 
+% adjust radDepth according to range shifter
+radDepths = radDepths + rangeShifter.eqThickness;
+
 % convert from MeV cm^2/g per primary to Gy mm^2 per 1e6 primaries
 conversionFactor = 1.6021766208e-02;
 
  % calculate initial focus sigma
 SigmaIni = matRad_interp1(baseData.initFocus.dist(focusIx,:)',baseData.initFocus.sigma(focusIx,:)',SSD);
 
+
+if exist('rangeShifter', 'var') && exist('radiationMode', 'var') &&  rangeShifter.eqThickness ~= 0
```

Comment:
hm, one could do that distinction in sigmaRashi, which would be in fact more natural. We can get rid of the existence checks as well, though I did it for downward compatibility of calcDoseBixel interface, but it's not really needed.